src/VPolytope.jl

```diff
@@ -218,9 +219,37 @@ function rand(::Type{VPolytope};
     return VPolytope(vertices)
 end
 
+"""
+    linear_map(M::AbstractMatrix{N}, P::VPolytope{N};
+               output_type::Type{<:LazySet}=VPolytope{N}) where {N<:Real}
 
-# --- AbstractPolytope interface functions ---
+Concrete linear map of a polytope in vertex representation.
+
+### Input
+
+- `M`           -- matrix
+- `P`           -- polytope in vertex representation
+- `output_type` -- (optional, default: `VPolytope`) type of the result
+
+### Output
+
+A set of type `output_type`.
 
+### Algorithm
+
+The linear map ``M`` is applied to each vertex of the given set ``P``, obtaining
+a polytope in V-representation.
+
+The default output is a `VPolytope`. If an `output_type` is given, the
+corresponding `convert` method is invoked.
+"""
+function linear_map(M::AbstractMatrix{N}, P::VPolytope{N};
```

Comment:
Can we have such a method for `VPolygon` as well?

Reply:
Yes, having a `linear_map` to dispatch on `VPolygon` should be cleaner. This is the purpose of #1133.